src/Fantomas.Tests/RecordTests.fs

```diff
@@ -1414,3 +1414,28 @@ module Foo =
     type private Bang =
         abstract Baz : int
 """
+
+
+[<Test>]
+let ``add space before paren in funcall, 1476`` () =
```

Comment:
I would go with ``add space before parenthesis when function identifier is SynExpr.Const, 1476``